index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins&display=swap"
+      rel="stylesheet"/>
+    <script
+      src="https://kit.fontawesome.com/79b3da0f3c.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="unit">
+        <header>
+            <nav class="navmenu">
+                <a href="">
+                    <p class="Logo">john Doe</p>
+                </a>
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+        </header>
+        <main class="main-frag">
+            <section class="headline-frag">
+                <div class="headline">
+                    <h1 class="headline1">Hey There.</h1>
+                    <h2 class="headline2">I am Alex</h2>
```

Comment:
- [ ] Kindly use a single `h1` tag for the main title. 😃 

## Figma design

![crr10](https://user-images.githubusercontent.com/103386183/198410963-9c878109-7d48-4ed0-9bc8-ffa65c27ae29.png)